public/app/plugins/datasource/prometheus/components/PromQueryField.tsx

```diff
@@ -77,6 +77,8 @@ export function willApplySuggestion(suggestion: string, { typeaheadContext, type
 interface PromQueryFieldProps extends ExploreQueryFieldProps<PrometheusDatasource, PromQuery, PromOptions> {
   history: Array<HistoryItem<PromQuery>>;
   ExtraFieldElement?: ReactNode;
+  placeholder?: string;
+  'data-testid'?: string;
```

Comment:
Do you know if there is there a better way of doing this? @domasx2 @torkelo @hugohaggmark

Reply:
No, don't think there is

Reply:
Ok, thanks 🙏🏻

Reply:
Shouldn't we always use `aria-label`? Then we get accessibility and testing in the same prop?